src/main/services/CaseService.ts

```diff
@@ -67,3 +70,61 @@ export const getPreloginCaseData = (redisClient: RedisClient, guid: string): Pro
     });
   });
 };
+
+export const formatCaseData = (fromApiCaseData: initiateCaseDraftResponse): CaseWithId => ({
+  id: fromApiCaseData.id,
+  state: fromApiCaseData.state,
+  isASingleClaim: fromApiCaseData.case_data.case_type === 'Single' ? YesOrNo.YES : YesOrNo.NO,
+});
+
+export class CaseApi {
+  constructor(private readonly axio: AxiosInstance) {}
+
+  public async getCase(caseTypeId = CcdDataModel.SINGLE_CASE_ENGLAND): Promise<CaseWithId | false> {
+    try {
+      const fetchedCases = await this.getDraftCases(caseTypeId);
+      switch (fetchedCases.length) {
```

Comment:
Is there any reason why switch is needed here? Maybe more readable to put it in a one liner like 
return fetchedCases.length === 0 ? false : formatCaseData(...)